plugins/dagsvisualizer/frontend/src/stores/BranchStore.tsx

```diff
@@ -105,6 +111,7 @@ export class BranchStore {
 
         b.isConfirmed = true;
         this.branches.set(confirmedBranch.ID, b);
+        updateConfirmedBranch(b, this.graph.cy);
```

Comment:
Pass  `this.graph` instead, and extract `cy` in `updateConfirmedBranch`.
We want to hide the graph library details in stores.